Server/api/userEmissions.js

```diff
@@ -29,17 +29,16 @@ router.get('/', async function (req, res, next) {
   weekCount = 0;
   monthCount = 0;
   retArr = [[0, 0, 0, 0, 0], // TODAY
-            [0, 0, 0, 0, 0], // YESTERDAY
-            [0, 0, 0, 0, 0], // 7 DAYS (1 WEEK)
-            [0, 0, 0, 0, 0]  // 31 DAYS (1 MONTH)
+  [0, 0, 0, 0, 0], // YESTERDAY
```

Comment:
Some of these changes aren't actually changes it's just my linter fixing formatting.